From f78a7db0e6fd6e759fb7af9897af51b25fd78cc2 Mon Sep 17 00:00:00 2001
From: Sergio Prado <sergio.prado@toradex.com>
Date: Wed, 11 Aug 2021 16:22:48 +0000
Subject: [PATCH 1/2] mbedtls: fix issue building with a newer GCC

This is a know issue reported in the community in [1], and can be fixed
with commit [2].

[1] https://github.com/ARMmbed/mbedtls/issues/4233
[2] https://github.com/ARMmbed/mbedtls/pull/4237/commits/2065a8d8af27c6cb1e40c9462b5933336dca7434

Signed-off-by: Sergio Prado <sergio.prado@toradex.com>
---
 lib/mbedtls-2.26.0/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/mbedtls-2.26.0/CMakeLists.txt b/lib/mbedtls-2.26.0/CMakeLists.txt
index 138adfe2d563..e79f2bdd1050 100644
--- a/lib/mbedtls-2.26.0/CMakeLists.txt
+++ b/lib/mbedtls-2.26.0/CMakeLists.txt
@@ -200,7 +200,7 @@ if(CMAKE_COMPILER_IS_GNU)
         endif()
     endif()
     if (GCC_VERSION VERSION_GREATER 7.0 OR GCC_VERSION VERSION_EQUAL 7.0)
-      set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wformat-overflow=2 -Wformat-truncation=2")
+      set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wformat-overflow=2 -Wformat-truncation")
     endif()
     set(CMAKE_C_FLAGS_RELEASE     "-O2")
     set(CMAKE_C_FLAGS_DEBUG       "-O0 -g3")
-- 
2.25.1

